libs/scripts/get_video_watching_durations.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import json
+import psycopg2
+
+
+# Gets datetime object from log 'time' string
+def parse_datetime(log_timestamp):
+    dt = log_timestamp.split("T")
+    date = dt[0].split("-")
+    time = dt[1].split(".")[0].split(":")
+    return datetime.datetime(int(date[0]), int(date[1]), int(date[2]), hour=int(time[0]), minute=int(time[1]), second=int(time[2]))
+
+
+# Prints total time of video-watching for every user (as a table) into console
```

Comment:
There is no need to print the result to stdout, because in common case there users amount is around 10 000 persons, so that it is impossible to look at stdout.